.ci/targets/macos_repo_checks.yaml

```diff
@@ -5,6 +5,10 @@ tasks:
   - name: update pods repo
     script: .ci/scripts/update_pods.sh
     infra_step: true # Note infra steps failing prevents "always" from running.
+  - name: Swift format
+    script: script/tool_runner.sh
+    args: ["format", "--fail-on-change", "--no-dart", "--no-clang-format", "--no-kotlin", "--no-java" ]
```

Comment:
It's probably worth a comment saying that these languages are done in `repo_checks.yaml`, so are excluded here to avoid duplication.

Reply:
Done.